src/doc/gui_manual/Operators/OperatorTypes/PICS/LCS_operator.rst

```diff
@@ -0,0 +1,183 @@
+.. _LCS operator:
+
+Lagrangian Coherent Structure operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The LCS operator utilizes Lyapunov Exponents based on the Cauchy-Green Tensor
+to highlight LCS's in vector fields. The Cauchy-Green tensor is used as an
+equivalent definition because it is independent of the mesh discretization. As
+such, when performing an FTLE calculation, the time can be specified as one
+would for a traditional FTLE, and the resulting value will be based on the
+maximal Eigen value.
+
+However, when performing the calculate with Finite Space Lyapunov Exponents
+(FSLE), instead of assuming a uniform mesh discretization and specifying the
+dispersion distance, we specify a dispersion factor which in a traditional FTLE
+is the dispersion distance divided by the initial distance. In the equivalent
+definition the dispersion distance is the maximal Eigen value. Thus when the
+maximal Eigen value is greater than the specified dispersion factor, then the
+exponent is calculated.
+
+For more details see the following paper: `A Comparison of Finite-Time and
+Finite-Size Lyapunov Exponents by Ronald Peikert, Armin Pobitzer, Filip Sadlo
+and Benjamin Schindler,
+<http://www.scivis.ethz.ch/publications/pdf/2014/peikert2014comparison.pdf>`_
+
+
+
+Source
+^^^^^^
+
+Source types
+""""""""""""
+
+The user can seed the integral curves using the native mesh or define their own
+rectilinear grid.
+   
+Native Mesh
+    Create seeds at the nodes of the native mesh. 
+
+Regular Grid
+    Create seeds at the nodes of a user-defined rectilinear grid. Parameters
+    are:
+
+    * Resolution - Three cartesian values separated with a space defining the
+      number of samples in each dimension.
+    * Data extents - Specify the starting and ending extents, using either the
+      native mesh's extents (Full) or user-defined extents (Subset).
+
+
+Auxilary Grid
+"""""""""""""
+
+When calculating the Jacobian for the Cauchy-Green tensor, one can use the
+neighboring points from the native mesh or one can specify an auxiliary grid
+that allows for the detection of finer features but at greater computational
+expense. Using an auxiliary grid is advantageous because it is independent of
```

Comment:
The user might like to know *how much* greater computational expense 2x, 5x, 10x? @ARSanderson can probably comment.